modules/aws_postgres/aws-postgres.md

```diff
@@ -26,6 +26,29 @@ Storage scaling is automatically managed by AWS Aurora, see the [official docume
 To add replicas to an existing cluser, follow the [official guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-replicas-adding.html).
 
 
+### Restoring from Snapshot
```

Comment:
I would add a note that opta doesn't support removing this value after the snapshot has been restored.